.github/workflows/nightly.yml

```
@@ -180,17 +180,7 @@ jobs:
         brew install create-dmg gettext
 
         # Remove conflicting brew items
-        brew remove --ignore-dependencies aom cairo composer curl fontconfig freetype gd ghostscript httpd imagemagick jpeg-xl libavif libheif libxft mongodb-community@5.0 mongosh node@16 openjpeg php r sox webp libpng brotli
-
-        # Delete fake harfbuzz if need be
-        if [ -f "/usr/local/lib/libharfbuzz.dylib" ]; then
-          sudo rm -r /usr/local/lib/libharfbuzz.dylib
-        fi
-
-        # And the .a for it as well
-        if [ -f "/usr/local/lib/libharfbuzz.a" ]; then
-          sudo rm -r /usr/local/lib/libharfbuzz.a
-        fi
+        brew remove --ignore-dependencies freetype harfbuzz libpng brotli
```

When I earlier tried making brew remove harfbuzz the CI failed with the error that harfbuzz was not installed.

---

see https://github.com/Inochi2D/inochi-creator/actions/runs/3179298771/jobs/5181643488

---

Hmm, I can't can't seem to reproduce it even with the exact same script https://github.com/tellowkrinkle/inochi-creator/actions/runs/3202454634/jobs/5231453673

I'm guessing it's GitHub messing with the package list in their CI image.  They seem to like to do that.  They recently un-preinstalled libjpeg which broke PCSX2's CI script for a bit.  One possibility is to `brew remove` each package separately and ignore errors with `|| true`, which is [what we now do in PCSX2](https://github.com/PCSX2/pcsx2/commit/8b375d66fae12022e3f5853d1a59dc39e98ae9f6).